app/models/broadcast_message.py

```diff
@@ -85,6 +85,17 @@ def create_from_content(cls, *, service_id, content, reference):
             )
         )
 
+    @classmethod
+    def update_from_content(cls, *, service_id, broadcast_message_id, content, reference):
+        broadcast_message_api_client.update_broadcast_message(
+            service_id=service_id,
+            broadcast_message_id=broadcast_message_id,
+            data={
+                "reference": reference,
+                "content": content,
+            },
+        )
```

Comment:
We should have a test to ensure that this gets called when appropriate.

Reply:
See new commits - added tests to make sure update/create is called appropriately